R/capture-requests.R

```diff
@@ -429,3 +434,41 @@ get_redactor <- function() {
 
   return(NULL)
 }
+
+#' Check dittodb environment path
+#'
+#' This function should generally not be used, but must be exported for the
+#' query recording function to work properly
+#'
+#' @param .dittodb_env Environment object
+#' @param n Integer. Type of error to throw.
+#'
+#' @return NULL, invisibly.
+#' @export
+check_db_path <- function(.dittodb_env, n = 1L) {
+  if(is.null(.dittodb_env$db_path)) {
+    if (n == 1L) {
+      msg <- c("Database capture failed",
+               "*" = "The database connection object was created before calling 'start_db_capturing()'",
+               "*" = "Please close the connection and ensure it's created after calling 'start_db_capturing()'.")
```

Comment:
```suggestion
               "*" = "Please ensure the connection is created after calling 'start_db_capturing()'.")
```

This is slightly more direct, and also shorter so hopefully that appeases the lintr

Reply:
i know rlang has that error feature, but i have not done it myself haha! i will do some research to see what that looks like, i forgot that was a thing! i will address the rest of the comments this week, thanks for the detailed comments :)

Reply:
updated!